test/utils/configuration-helper.js

```diff
@@ -1,9 +1,48 @@
-const { CIRCLECI } = process.env;
+const { CIRCLECI, JE_ENV, TEST_TYPE } = process.env;
+const chromeSettings = require('../configuration/chrome/chrome.settings').default();
+const browserstackSettings = require('../configuration/browserstack/browserstack.settings').default();
+const environmentSettings = CIRCLECI ? require('../configuration/ci.test-settings').default()
+: require('../configuration/local.test-settings').default()
 
-exports.setTestSettings = () => {
 
-    const testSettings = CIRCLECI ? require('../configuration/ci.test-settings').default()
-                                    : require('../configuration/local.test-settings').default()
+exports.getConfigurationSettings = () => {
 
-    return testSettings;
+    let runnerSettings = null;
+    let testCapabilities = null;
```

Comment:
No need to initialise as `null`, if `undefined` works for you as well, so you can do `let runnerSettings;`.